models/issue_reaction.go

```diff
@@ -39,10 +40,28 @@ func (opts *FindReactionsOptions) toConds() builder.Cond {
 	}
 	if opts.CommentID > 0 {
 		cond = cond.And(builder.Eq{"reaction.comment_id": opts.CommentID})
+	} else if opts.CommentID == -1 {
```

Comment:
Any comment about the `-1`?

Reply:
Is 314219850edbab9b27cf940edb50151cb9f19d2e good enouth ?